frontend/src/views/WheresMyGeneV2/components/HeatMap/components/YAxisChart/index.tsx

```diff
@@ -265,35 +262,33 @@ const CellTypeButton = ({
           </Tooltip>
         </CellTypeLabelStyle>
 
-        {!FMG_EXCLUDE_TISSUES.includes(tissue) &&
-          cellType &&
-          cellType.total_count > 25 &&
-          cellType.optionId === COMPARE_OPTION_ID_FOR_AGGREGATED && (
-            <InfoButtonWrapper
-              className={EXCLUDE_IN_SCREENSHOT_CLASS_NAME}
-              onClick={() => {
-                if (cellType) {
-                  generateMarkerGenes(cellType, tissueID);
-                  track(EVENTS.WMG_FMG_INFO_CLICKED, {
-                    combination: `${cellType.name}, ${tissue}}`,
-                  });
-                }
-              }}
-            >
-              <StyledImage
-                data-testid="marker-gene-button"
-                src={InfoSVG.src}
-                /**
-                 * (thuang): https://nextjs.org/docs/pages/api-reference/components/image-legacy#layout
-                 * Use the <StyledImage /> width and height, since default is `intrinsic`
-                 */
-                layout="fixed"
-                width="10"
-                height="10"
-                alt={`display marker genes for ${cellType.name}`}
-              />
-            </InfoButtonWrapper>
-          )}
+        {cellType && cellType.optionId === COMPARE_OPTION_ID_FOR_AGGREGATED && (
```

Comment:
here, the logic is just updated to always display the info icon button. most of the line changes are just whitespace